pipeline/compressors/__init__.py

```diff
@@ -142,7 +142,11 @@ def reconstruct(match):
 
     def concatenate(self, paths):
         """Concatenate together a list of files"""
-        return "\n".join([self.read_text(path) for path in paths])
+        # Note how a semicolon is added between the two files to make sure that
+        # their behavior is not changed. '(expression1)\n(expression2)' calls
+        # `expression1` with `expression2` as an argument! Superfluos semicolons
+        # are valid in JavaScript and will be removed by the minifier.
+        return "\n;".join([self.read_text(path) for path in paths])
```

Comment:
The one-letter change in this single line is the only change made to the code in this PR. The rest are just comments and tests updates.